test/test_optim.py

```diff
@@ -1294,6 +1299,8 @@ def closure():
             return closure_loss if optim_info.step_requires_closure else None
 
         for optim_input in cpu_optim_inputs:
+            if "fused" in optim_input.kwargs and optim_input.kwargs["fused"]:
+                self.skipTest(f"cuda is not supported for fused on {optim_cls.__name__}")
```

Comment:
This message does not seem to match the conditional

Reply:
Thanks for advice, changed it.